tests/src/runner/podmanDesktopRunner.ts

```diff
@@ -0,0 +1,81 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ElectronApplication, Page } from 'playwright';
+import { _electron as electron } from 'playwright';
+
+export class PodmanDesktopRunner {
+  private _path: string;
+  private _running: boolean;
+  private _properties: object;
+  private _app!: ElectronApplication;
+
+  constructor(path: string, properties: object) {
+    this._path = path;
+    this._running = false;
+    this._properties = properties;
+  }
+
+  public async startApp(): Promise<ElectronApplication> {
```

Comment:
can be just start and return by default a page instead of the electron application (can have a getElectron() method for this) as most of playwright tests will use the page